model/incoming_webhook.go

```diff
@@ -93,8 +93,8 @@ func (o *IncomingWebhook) IsValid() *AppError {
 		return NewAppError("IncomingWebhook.IsValid", "model.incoming_hook.display_name.app_error", nil, "", http.StatusBadRequest)
 	}
 
-	if len(o.Description) > 128 {
-		return NewAppError("IncomingWebhook.IsValid", "model.incoming_hook.description.app_error", nil, "", http.StatusBadRequest)
+	if len(o.Description) > 500 {
```

Comment:
I removed error because Jira ticket says `adjust the incoming webhooks and outgoing webhooks listing interfaces to truncate the description to fit the available space.` - I thought that we don't want  to throw errors anymore and just need to truncate everything to 500

Reply:
I think the ticket is a bit unclear about that. Usually, we have the web app truncate the string (like it does in your other PR), but the server just returns an error if someone uses the API. We do this because we don't want to surprise the user by truncating the string without telling them. They know it has been truncated in the UI because the textbox doesn't let them type more than 500 characters, but when they're using the API, they have no way of knowing that it's too long unless we return an error.

Reply:
Yeah, that makes sense now. I added API errors back and thanks for the explanation!